src/pres/loader/PGNLoader.js

```diff
@@ -16,36 +16,39 @@ export default class PGNLoader extends React.Component {
     constructor(props) {
         super(props)
         let selectedSite = new URLSearchParams(window.location.search).get("source")
+        let playerName = new URLSearchParams(window.location.search).get("playerName")
```

Comment:
Oops, I should probably pull `new URLSearchParams(window.location.search)` out into a variable :)

Reply:
(done)